pkg/buildkit/buildkit.go

```diff
@@ -334,3 +346,101 @@ func WithFileString(s *llb.State, path, contents string) llb.State {
 func WithFileBytes(s *llb.State, path string, contents []byte) llb.State {
 	return s.File(llb.Mkfile(path, 0o600, contents))
 }
+
+func QemuAvailable(p *types.PatchPlatform) bool {
+	if p == nil {
```

Comment:
do you think this is a func we could add test for?

Reply:
I've added some tests for this with mocking for the os apis